CorsixTH/Src/th_movie.h

```diff
@@ -331,6 +331,12 @@ class movie_player {
   //! Stop the currently playing movie
   void stop();
 
+  //! Pause the currently playing movie
+  //!
+  //! This method is not thread safe, it should only be called from the event
+  //! thread.
```

Comment:
Could you maybe align this comment to use wording within the function itself. You use 'close to thread safe' but then say 'not' here (arguably both are true, but infer differences). You could point back to the function for deeper explanation, I'd already be going "why?" reading this anyway.